ansible/deploy_aws.yml

```diff
@@ -0,0 +1,24 @@
+- name: provision aws resources
+  hosts: localhost
+
+  tasks:
+    - name: create bucket
+      s3: bucket={{airflow_bucket}} region={{region}} mode=create
+
+    - name: copy EMR bootstrap script
+      s3: bucket={{airflow_bucket}} region={{region}} object=steps/airflow.sh src=ansible/files/spark/airflow.sh mode=put
+
+    - name: create task definition
+      shell: "{{ item }}"
+      with_items:
+        - ecs-cli configure --cluster {{ ecs_cluster_name }}
+        - ecs-cli compose --project-name airflow-test --file ansible/files/docker-compose.yml up
```

Comment:
What does the `project-name` argument do? Can we use something more descriptive than `airflow-test`?